pkg/asset/installconfig/azure/validation.go

```diff
@@ -127,7 +129,7 @@ func ValidateInstanceType(client API, fieldPath *field.Path, region, instanceTyp
 				errMsg := fmt.Sprintf("PremiumIO not supported for instance type %s", instanceType)
 				allErrs = append(allErrs, field.Invalid(fieldPath.Child("osDisk", "diskType"), diskType, errMsg))
 			}
-		} else if strings.EqualFold(*capability.Name, "UltraSSDAvailable") {
+		} else if strings.EqualFold(*capability.Name, ultraSSDCapability) {
```

Comment:
```suggestion
		} else if strings.EqualFold(*capability.Name, aztypes.AcceleratedNetworkingEnabled) {
```
This should be fixed by either #5840 or more probably #5846, although of course you are free to fix here.

Reply:
You mean I should add `UltraSSDAvailable` as a type in `aztypes`?